src/settings/Settings.tsx

```diff
@@ -270,6 +270,41 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         // Reload to ensure that the left panel etc. get remounted
         controller: new ReloadOnChangeController(),
     },
+    "feature_disableAEC": {
```

Comment:
These probably shouldn't be features but rather settings

Reply:
I had a small debate with myself over where to put them, they are settings but also closely tied to the video rooms, that's why I put them in there. But I can resettle them if that is better.